File: src/actions/HelloWorldAction.java
package actions;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.ui.Messages;

public class HelloWorldAction extends AnAction {

    @Override
    public void actionPerformed(AnActionEvent anActionEvent) {
        Messages.showInfoMessage("Hello World!", "idea demo plugin");
    }

}

File: src/main/kotlin/actions/HelloWorldAction.kt
package actions

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.ui.Messages

class HelloWorldAction : AnAction() {

    override fun actionPerformed(anActionEvent: AnActionEvent) {
        Messages.showInfoMessage("Hello World!", "idea demo plugin")
    }

}
